### PR TITLE
Design v3 style update

### PR DESCRIPTION
**Pull request by shaunanoordin**
## PR Overview
* This PR updates the visual look (colours, fonts, layout, etc) to match the latest (v3) design guide.
* This is mostly a design placeholder; functionality is not being built yet.
* We are still missing a shared Zooniverse header, but this should be coming from the Zooniverse-React-Components library.

### Visual Reference
screen shot 2017-06-05 at 16 49 10
https://indd.adobe.com/view/e2b8cf36-eb64-4e7b-8cf0-c923214d2e67

### CODE REVIEW

**Comment by shaunanoordin** [comment]
* We now have a placeholder `/classify` page, and this is how it looks like:
screen shot 2017-06-05 at 17 37 06

**Comment by shaunanoordin** [comment]
Update:
screen shot 2017-06-07 at 16 37 45

**Comment by shaunanoordin** [comment]
OK we're done here.

screen shot 2017-06-09 at 13 34 26

Ready for review!

File: src/containers/ClassifierContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+//import { checkLoginUser, loginToPanoptes, logoutFromPanoptes } from '../ducks/login';
```
Comment (wgranger) [inline src/containers/ClassifierContainer.jsx]: I take it these are the ducks that still need to be written for classifier functionality.

**Comment by wgranger** [comment]
This looks good overall and is quite succinct. A good bit was done with a few lines. The next step seems to be getting those ducks in order for api calls. Something we need to think of in the future is, again, making it mobile-friendly, but that is not a concern at this point and can prob be done in a separate PR later with a few lines of code. Overall, this provides a nice placeholder for components to come.

I like the use of items like `rem` and `<i>` in this PR - something we don't use too often in PFE. 

I've tested this out in all browsers, and everything appears to work well. I was unable to get IE up and running on virtual box, but that shouldn't hold up this PR.

File: src/containers/ClassifierContainer.jsx (hunk 1 above)
Comment (shaunanoordin) [inline src/containers/ClassifierContainer.jsx]: Quack, quack quack quack. (Yes, that is correct.)

**Pull request merged**